docs/release/k3s-release.md

```diff
@@ -0,0 +1,253 @@
+git push $REMOTE staging/src/k8s.io/cri-api/v1.22.12-k3s1
+git push $REMOTE staging/src/k8s.io/csi-translation-lib/v1.22.12-k3s1
+git push $REMOTE staging/src/k8s.io/kube-aggregator/v1.22.12-k3s1
+git push $REMOTE staging/src/k8s.io/kube-controller-manager/v1.22.12-k3s1
+git push $REMOTE staging/src/k8s.io/kubectl/v1.22.12-k3s1
+git push $REMOTE staging/src/k8s.io/kubelet/v1.22.12-k3s1
+git push $REMOTE staging/src/k8s.io/kube-proxy/v1.22.12-k3s1
+git push $REMOTE staging/src/k8s.io/kube-scheduler/v1.22.12-k3s1
+git push $REMOTE staging/src/k8s.io/legacy-cloud-providers/v1.22.12-k3s1
+git push $REMOTE staging/src/k8s.io/metrics/v1.22.12-k3s1
+git push $REMOTE staging/src/k8s.io/sample-apiserver/v1.22.12-k3s1
+git push $REMOTE staging/src/k8s.io/sample-cli-plugin/v1.22.12-k3s1
+git push $REMOTE staging/src/k8s.io/sample-controller/v1.22.12-k3s1
+git push $REMOTE v1.22.12-k3s1
+```
+## Push tags to k3s-io remote
+```
+export REMOTE=k3s-io
+./push.sh
+```
+## Updating k3s with the new tags
+You now have a collection of tagged kubernetes modules in your worktree. By updating go.mod in k3s to point at these modules we will then be prepared to open a PR for review.
+```sh
+cd $GOPATH/src/github.com/rancher/k3s
+git remote add upstream https://github.com/k3s-io/k3s.git
+git fetch upstream
+git checkout -B $NEW_K3S_VER upstream/$RELEASE_BRANCH
+git clean -xfd
+ 
+sed -Ei "\|github.com/k3s-io/kubernetes| s|${OLD_K3S_VER}|${NEW_K3S_VER}|" go.mod
+sed -Ei "s/k8s.io\/kubernetes v\S+/k8s.io\/kubernetes $NEW_K8S/" go.mod
+sed -Ei "s/$OLD_K8S_CLIENT/$NEW_K8S_CLIENT/g" go.mod 
+ 
+# since drone perform the builds and tests for the updated tags we no longer need to run make locally.
+# We now update the go.sum by running go mod tidy:
+go mod tidy
+git add go.mod go.sum
+
+git commit --all --signoff -m "Update to $NEW_K8S"
+git push --set-upstream origin $NEW_K3S_VER 
+```
+
+Create a commit with all the changes, and push this upstream.
+Create a PR to merge your branch into the corresponding release branch, and wait for CI to run tests on the PR. Make sure to create the PR against the associated release branch for this update.
+
+Once CI passes and you receive two approvals, you may now squash-merge the PR and then tag an RC.
+
+# Create a Release Candidate 
+Releases are kicked off and created by tagging a new tag.
+To create a new release in Github UI perform the following:
+
+1. Set title and tag as v1.22.5-rc1+k3s1, target as release-1.22
+2. Leave description blank.
+3. Check the pre-release field.
+4. Publish
+
+The resulting run can be viewed here: 
+[k3s-io/k3s Drone Dashboard](https://drone-publish.k3s.io/k3s-io/k3s)
+
+# Create GA Release Candidate
+Once QA has verified after 24-72 hours that the RC is good (or that any fixes have been added in follow up RC candidates), it is time for the general release.
+
+1. Create new release in GH UI
+2. Set title: v1.22.5+k3s1, add description with release notes. Leave the tag section blank.
```

Comment:
Update to be more like the comment above.

Reply:
This is done.

Reply:
Was the change pushed?